src/main/java/sql/Main.java

```diff
@@ -0,0 +1,15 @@
+package sql;
+
+import additional.DataOperations;
+import additional.DatabaseConnector;
+import model.Tournament;
+
+public class Main {
```

Comment:
Make it as separte PR.
It is very hard to understand what is what in single PR.